src/Database/Adapter/SQLite.php

```
@@ -513,6 +603,47 @@ public function deleteIndex(string $collection, string $id): bool
         }
     }
 
+    /**
+     * Drop the FTS5 virtual table (and its triggers) corresponding to the
+     * fulltext index `$id` on `$collection`, if one exists. Returns true if
+     * a matching table was found and dropped.
+     */
+    protected function dropFulltextIndexById(string $collection, string $id): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)